scripts/k8s-deployment-sim.sh

```diff
@@ -1,5 +1,20 @@
 #!/bin/bash
 
+printf "\n\n\n*** [0] install server with tls template and HA in k8s\n"
```

Comment:
suggestion "..install HA gateway template in k8s.."

Reply:
updated